client/commands/query/query_test.go

```diff
@@ -0,0 +1,140 @@
+package query
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/basecoin/app"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/go-wire"
+	lc "github.com/tendermint/light-client"
+	nm "github.com/tendermint/tendermint/node"
+	"github.com/tendermint/tendermint/rpc/client"
+	rpctest "github.com/tendermint/tendermint/rpc/test"
+	"github.com/tendermint/tmlibs/log"
+	ctest "github.com/tendermint/tmlibs/test"
+)
+
+var node *nm.Node
+
+func getLocalClient() client.Local {
+	return client.NewLocal(node)
+}
+
+func TestMain(m *testing.M) {
+	// start a tendermint node (and merkleeyes) in the background to test against
+	logger := log.TestingLogger()
+	store, err := app.NewStore("", 0, logger)
+	if err != nil {
+		panic(err)
+	}
+	app := app.NewBasecoin(etc.NewHandler(), store, logger)
```

Comment:
never mind, you are not running our query commands, and passing in the raw key.  perfect like this.